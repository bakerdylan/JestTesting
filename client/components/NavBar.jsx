import React from 'react';
import { Navbar } from 'react-bootstrap';

export default () => (
  <Navbar collapseOnSelect fixedTop>
    <Navbar.Header>
      <Navbar.Brand>
        Jest Testing
      </Navbar.Brand>
    </Navbar.Header>
  </Navbar>
);
