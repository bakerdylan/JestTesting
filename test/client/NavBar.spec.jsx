import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import NavBar from '../../client/components/NavBar';

describe('<NavBar />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<NavBar />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
