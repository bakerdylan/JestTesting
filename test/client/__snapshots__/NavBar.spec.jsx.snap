// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> renders correctly 1`] = `
Array [
  <Uncontrolled(Navbar)
    bsStyle="default"
    collapseOnSelect={true}
    fixedTop={true}
>
    <NavbarHeader>
        <NavbarBrand>
            <a
                href="/"
            >
                Jest Testing
            </a>
        </NavbarBrand>
    </NavbarHeader>
</Uncontrolled(Navbar)>,
]
`;
